### PR TITLE
Incorporating changes made by NCO SPA

### PR DESCRIPTION
**Pull request by ShelleyMelchior-NOAA**
Changes address modification of output com dir for mods and ancillary tweaks to cdas and global job scripts.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
Changes were tested in local cron processes.
SPA generated output in  `/lfs/h1/ops/test/com/obsproc/v1.0/mods.2022` looks correct.

File: jobs/JOBSPROC_CDAS_DUMP [hunk 1]
```diff
@@ -161,7 +161,7 @@ export FIXbufr=${FIXbufr:-$HOMEbufr_dump/fix}
 # TANK is root directory path to observational database used in dumpjb
 #  and engice, sst & snowdepth grib files used in model script
 # --------------------------------------------------------------------
-export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/${envir}/dcom}
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/prod/dcom}
```
Comment (ilianagenkova) [inline jobs/JOBSPROC_CDAS_DUMP:164]: Are NCO stepping away from ${envir}?

File: jobs/JOBSPROC_MODS [hunk 2]
```diff
@@ -169,15 +169,17 @@ cp $UTILROOT/ush/setpdy.sh .; chmod +x setpdy.sh
 ##############################
 # Run setpdy and initialize PDY variables
 ##############################
-setpdy.sh
+setpdy.sh 9
```
Comment (ilianagenkova) [inline jobs/JOBSPROC_MODS:172]: Do we need this "9"? I think it's leftover from Xiaoxue's testing.

**Review by ilianagenkova — CHANGES_REQUESTED** [review]
jobs/JOBSPROC_MODS has a change I am not sure about - line 172 , is the **9** needed?

File: jobs/JOBSPROC_MODS (hunk 2 above)
Comment (ShelleyMelchior-NOAA) [reply jobs/JOBSPROC_MODS:172]: Yes, it is needed. `setpdy.sh` with no parameters will default to setting PDYm and PDYp back (m for minus) 7 days and forward (p for plus) 7 days.  Passing in the parameter will override the default of 7 and calculate backward and forward 9 days. The variable PDYm9 is used 2 lines lower to help set the new MODSDIR location.

File: jobs/JOBSPROC_CDAS_DUMP (hunk 1 above)
Comment (ShelleyMelchior-NOAA) [reply jobs/JOBSPROC_CDAS_DUMP:164]: Yes, in the context of DCOMROOT they are. DCOMROOT does not have a para or test area. Instead it is prod or dev, which are explicitly specified.

**Review by ilianagenkova — APPROVED** [review]

**Pull request merged**